### ISSUE TITLE
[Serial]Storage Starting a VirtualMachineInstance [rfe_id:2288][crit:high][vendor:cnv-qe@redhat.com][level:component] With Cirros BlockMode PVC [test_id:1015] should be successfully started

### ISSUE THREAD

**Issue by dhiller** [kubevirt/kubevirt#4314]
**Is this a BUG REPORT or FEATURE REQUEST?**:

> Uncomment only one, leave it on its own line:
>
/kind bug
> /kind enhancement

**What happened**:
```
tests/storage_test.go:796
Cirros login successfully
Unexpected error:
    <*kubecli.AsyncSubresourceError | 0xc0036a0c60>: {
        err: "Can't connect to websocket (500): websocket: bad handshake\n",
        StatusCode: 500,
    }
    Can't connect to websocket (500): websocket: bad handshake
    
occurred
tests/storage_test.go:807
```

Source: https://prow.apps.ovirt.org/view/gcs/kubevirt-prow/pr-logs/pull/kubevirt_kubevirt/4277/pull-kubevirt-e2e-k8s-1.17/1312368188956086273

**What you expected to happen**:

**How to reproduce it (as minimally and precisely as possible)**:

**Anything else we need to know?**:

**Environment**:
- KubeVirt version (use `virtctl version`):
- Kubernetes version (use `kubectl version`):
- VM or VMI specifications:
- Cloud provider or hardware configuration:
- OS (e.g. from /etc/os-release):
- Kernel (e.g. `uname -a`):
- Install tools:
- Others:

**Comment by rhrazdil** [comment]
Happened again here:
https://prow.apps.ovirt.org/view/gcs/kubevirt-prow/pr-logs/pull/kubevirt_kubevirt/4152/pull-kubevirt-e2e-k8s-1.16/1313160032598102016

**Comment by akrejcir** [comment]
Happened again:
https://prow.apps.ovirt.org/view/gcs/kubevirt-prow/pr-logs/pull/kubevirt_kubevirt/4415/pull-kubevirt-e2e-k8s-1.18/1321250844049412096

**Comment by rmohr** [comment]
/close

"Fixed" in #4522. It takes sometimes unusually long until the kubelet updates the certificate secrets.
